src/flatpakUtils.ts

```diff
@@ -1,46 +1,30 @@
 import { Command } from './command'
 
-export interface Runtime {
+export interface FlatpakEntry {
     id: string
     version: string
 }
 
-export function getAvailableRuntimes(): Runtime[] {
-    const command = new Command('flatpak', ['list', '--runtime', '--columns=application,branch'])
+/**
+ * Retrieves the list of installed flatpak applications or runtimes.
+ * @param type applications or runtimes
+ */
+export function getAvailable(type: 'app' | 'runtime'): FlatpakEntry[] {
```

Comment:
Should the type parameter be dropped, since `app` variant is currently never used?

Reply:
Would keep it as is for now